src/gobprepare/cloner/oracle_to_postgres.py

```diff
@@ -158,6 +159,60 @@ def _copy_data(self) -> int:
 
         return rows_copied
 
+    def _list_to_chunks(self, lst: list, chunk_size: int):
+        """Divides a list in chunks of chunk_size. Returns a list of dictionaries with "min" and "max" as keys, where
+        "min" is inclusive, and "max" is exclusive.
```

Comment:
Nice, as our Dutch Edsger Dijkstra already wrote in 1982,
https://www.cs.utexas.edu/users/EWD/ewd08xx/EWD831.PDF